test/utils.sh

```diff
@@ -1108,12 +1108,6 @@ get_bundle_suggested_namespace() {
     | select(.type == "olm.csv.metadata") 
     | .value.annotations["operatorframework.io/suggested-namespace"]')
```

Comment:
I recently encountered a CSV where .value is not always an `object`. In that case, this jq query will fail. I know it's not in the scope of this MR, but it might be worth fixing this too while you're at it.

Reference: https://github.com/konflux-ci/build-definitions/commit/ec0cc5d92a679e45914dc7ef6b1b28d7885fe365

Reply:
It's ok if you want to address this in a separate PR. But this is definitely worth addressing